services/workflows-service/src/workflow/dtos/find-workflows-list.dto.ts

```diff
@@ -66,6 +73,7 @@ export const FindWorkflowsListSchema = z.object({
         .array(z.union([z.literal('').transform(() => null), z.string().nonempty()]))
         .optional(),
       status: z.array(z.nativeEnum(WorkflowRuntimeDataStatus)).optional(),
+      caseStatus: z.array(z.string()).optional(),
```

Comment:
Do we need `caseStatus` each page? If so, maybe it should be named `state` or `caseState` for parity sake.